src/unittest/test_servermodmanager.cpp

```diff
@@ -41,6 +45,9 @@ class TestServerModManager : public TestBase
 	void testGetModNames();
 	void testGetModMediaPathsWrongDir();
 	void testGetModMediaPaths();
+
+private:
+	std::unique_ptr<MetricsBackend> m_test_metrics_backend;
```

Comment:
This could just be a `MetricsBackend m_test_metrics_backend`

Reply:
exact